src/git-command-manager.ts

```diff
@@ -11,7 +11,8 @@ import {GitVersion} from './git-version'
 
 // Auth header not supported before 2.9
 // Wire protocol v2 not supported before 2.18
-export const MinimumGitVersion = new GitVersion('2.18')
+// sparse-checkout not supported before 2.25
+export const MinimumGitVersion = new GitVersion('2.25')
```

Comment:
see https://git-scm.com/docs/git-sparse-checkout/2.25.0
and note that https://git-scm.com/docs/git-sparse-checkout/2.24.0 results in a redirect to https://git-scm.com/docs/git-sparse-checkout

Reply:
Correction:  minimum git version should be `2.28`  (see #1386)